i18n/en-US.properties

```diff
@@ -110,6 +110,10 @@ be.errorOpenWithDescription = Opening this file with other services is currently
 be.errorSkill = Error
 # Message shown when there is an error.
 be.errorState = A network error has occurred while trying to load.
+# Header message to the user when an Open With integration fails to execute
+be.executeIntegrationOpenWithErrorHeader = We're sorry, this integration is currently unavailable.
```

Comment:
' -> ’
accented

There are a couple of others I see too, maybe search/replace